packages/lesswrong/server/migrations/20231228T160733.add_wrapped2023Viewed_to_users.ts

```diff
@@ -1,12 +1,13 @@
 export const acceptsSchemaHash = "2c8fcbdc99caee38d7f728acc6428352";
 
+import { BoolType } from "../../lib/sql/Type";
 import Users from "../../lib/vulcan-users";
-import { addField, dropField } from "./meta/utils";
+import { addRemovedField, dropRemovedField } from "./meta/utils";
 
 export const up = async ({db}: MigrationContext) => {
-  await addField(db, Users, "wrapped2023Viewed");
+  await addRemovedField(db, Users, "wrapped2023Viewed", new BoolType())
```

Comment:
If we wanted to be really accurate here the type could be `new DefaultValueType(new BoolType(), false)`, but it doesn't really matter.